test/edu/ucsb/nceas/MCTestCase.java

```diff
@@ -405,165 +360,164 @@ protected String getTestEmlDoc(String title, String emlVersion) {
         String testDocument = "";
 
         String header;
-		if (emlVersion == EML2_0_1) {
-			header = testEml_201_Header;
-		} else if (emlVersion == EML2_1_0) {
-			header = testEml_210_Header;
-		} else { // if (emlVersion == EML2_1_1) {
-			header = testEml_211_Header;
-		}
-        
+        if (emlVersion == EML2_0_1) {
+            header = testEml_201_Header;
+        } else if (emlVersion == EML2_1_0) {
+            header = testEml_210_Header;
+        } else { // if (emlVersion == EML2_1_1) {
+            header = testEml_211_Header;
+        }
```

Comment:
Is this missing an EML 2.2.0 case? Big changes from 2.1.1. It's a bummer our EML versions are hardcoded in the java test code. Don't go down this rabbit hole, but for future reference we should try to not hardcode stuff like this.

Reply:
(Agreed and noted. Not familiar with this file at all - only edits I made to this file were in the static init block up top, to change how properties are loaded for tests at runtime)